As it stands, the build of this is failing:

```
In file included from /usr/include/stdio.h:27:0,
                 from /build_ldc2/parts/ldc-bootstrap/src/runtime/phobos/etc/c/zlib/gzguts.h:21,
                 from /build_ldc2/parts/ldc-bootstrap/src/runtime/phobos/etc/c/zlib/gzclose.c:6:
/usr/include/features.h:367:25: fatal error: sys/cdefs.h: No such file or directory
compilation terminated.
runtime/CMakeFiles/phobos2-ldc_32.dir/build.make:1682: recipe for target 'runtime/CMakeFiles/phobos2-ldc_32.dir/phobos/etc/c/zlib/gzclose.c.o' failed
make[2]: *** [runtime/CMakeFiles/phobos2-ldc_32.dir/phobos/etc/c/zlib/gzclose.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
In file included from /usr/include/limits.h:25:0,
                 from /usr/lib/gcc/x86_64-linux-gnu/5/include-fixed/limits.h:168,
                 from /usr/lib/gcc/x86_64-linux-gnu/5/include-fixed/syslimits.h:7,
                 from /usr/lib/gcc/x86_64-linux-gnu/5/include-fixed/limits.h:34,
                 from /build_ldc2/parts/ldc-bootstrap/src/runtime/phobos/etc/c/zlib/zconf.h:395,
                 from /build_ldc2/parts/ldc-bootstrap/src/runtime/phobos/etc/c/zlib/zlib.h:34,
                 from /build_ldc2/parts/ldc-bootstrap/src/runtime/phobos/etc/c/zlib/zutil.h:22,
                 from /build_ldc2/parts/ldc-bootstrap/src/runtime/phobos/etc/c/zlib/deflate.h:16,
                 from /build_ldc2/parts/ldc-bootstrap/src/runtime/phobos/etc/c/zlib/trees.c:37:
/usr/include/features.h:367:25: fatal error: sys/cdefs.h: No such file or directory
compilation terminated.
In file included from /usr/include/limits.h:25:0,
                 from /usr/lib/gcc/x86_64-linux-gnu/5/include-fixed/limits.h:168,
                 from /usr/lib/gcc/x86_64-linux-gnu/5/include-fixed/syslimits.h:7,
                 from /usr/lib/gcc/x86_64-linux-gnu/5/include-fixed/limits.h:34,
                 from /build_ldc2/parts/ldc-bootstrap/src/runtime/phobos/etc/c/zlib/zconf.h:395,
                 from /build_ldc2/parts/ldc-bootstrap/src/runtime/phobos/etc/c/zlib/zlib.h:34,
                 from /build_ldc2/parts/ldc-bootstrap/src/runtime/phobos/etc/c/zlib/zutil.h:22,
                 from /build_ldc2/parts/ldc-bootstrap/src/runtime/phobos/etc/c/zlib/infback.c:13:
/usr/include/features.h:367:25: fatal error: sys/cdefs.h: No such file or directory
compilation terminated.
runtime/CMakeFiles/phobos2-ldc_32.dir/build.make:1634: recipe for target 'runtime/CMakeFiles/phobos2-ldc_32.dir/phobos/etc/c/zlib/trees.c.o' failed
make[2]: *** [runtime/CMakeFiles/phobos2-ldc_32.dir/phobos/etc/c/zlib/trees.c.o] Error 1
runtime/CMakeFiles/phobos2-ldc_32.dir/build.make:1610: recipe for target 'runtime/CMakeFiles/phobos2-ldc_32.dir/phobos/etc/c/zlib/infback.c.o' failed
make[2]: *** [runtime/CMakeFiles/phobos2-ldc_32.dir/phobos/etc/c/zlib/infback.c.o] Error 1
```

I presume this is down to an absence of 32-bit dependencies for druntime/phobos.  I'm going to try adding `lib32c-dev` and `lib64c-dev` as build dependencies.

--- a/ldc-config/ldc2.conf
+++ b/ldc-config/ldc2.conf
@@ -10,6 +10,9 @@ default:
         "-I%%ldcbinarypath%%/../include/d/ldc",
         "-I%%ldcbinarypath%%/../include/d",
         "-L-L%%ldcbinarypath%%/../lib",
+        "-L-L%%ldcbinarypath%%/../lib32",
+        "-L-L%%ldcbinarypath%%/../lib64",
+        "-L--no-warn-search-mismatch",
         "-defaultlib=phobos2-ldc,druntime-ldc",
         "-debuglib=phobos2-ldc-debug,druntime-ldc-debug"
     ];

--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -35,11 +35,12 @@ parts:
     - -DCMAKE_C_FLAGS_RELEASE='-O3 -DNDEBUG -Wa,-mrelax-relocations=no'
     - -DLLVM_ROOT_DIR=../../llvm/install
     - -DLDC_INSTALL_LTOPLUGIN=ON
+    - -DMULTILIB=ON
     - -DCMAKE_VERBOSE_MAKEFILE=1
     stage:
     - -etc/ldc2.conf
     build-packages:
-    - gcc
+    - gcc-multilib
     - libconfig++-dev
     - libedit-dev
     - zlib1g-dev
@@ -62,12 +63,13 @@ parts:
     configflags:
     - -DCMAKE_BUILD_TYPE=Release
     - -DLLVM_ROOT_DIR=../../llvm/install
+    - -DMULTILIB=ON
     stage:
     - -*
     prime:
     - -*
     build-packages:
-    - g++
+    - g++-multilib
     - libconfig++-dev
     - libedit-dev
     - zlib1g-dev

